doc/cdi-operator-olm.md

```diff
@@ -1,8 +1,18 @@
-# OLM (Operator Lifecycle Management) intergartion
+# OLM (Operator Lifecycle Management) CDI (Containerized Data Importer) intergartion
+
+## Table of Contents 
+[OLM Overview](#overview)
```

Comment:
Can you make this a list with
```
 * [OLM Overview](#overview)
```
If you view the document, everything is on a single line. You can view the file by clicking the ... on the top right, and selecting view file.

Reply:
Done